src/ontology/pato-edit.obo

```diff
@@ -4118,25 +4118,25 @@ is_a: PATO:0001598 ! protruding
 id: PATO:0000644
 name: hyperplastic
 alt_id: PATO:0000943
-def: "A increased size quality inhering in an organ or tissue by virtue of the bearer's exhibiting increased number of cells." [Wikipedia:http\://en.wikipedia.org/wiki/Hyperplastic]
+def: "A physical quality inhering in an organ or tissue by virtue of the bearer's exhibiting an increased number of cells, typically associated with an increase in size." [Wikipedia:Hyperplasia]
```

Comment:
```suggestion
def: "A quality inhering in an organ or tissue by virtue of the bearer's exhibiting an increased number of cells, typically associated with an increase in size." [Wikipedia:Hyperplasia]
```
Is *'physical'* necessary in the definiton? 'hyperplastic' is not a child of [PATO:0001241 physical object quality](https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0001241&lang=en&viewMode=PreferredRoots&siblings=false).

Reply:
yes it is...
but I guess we don't need to specify it in the definition if people object(?)